Formula/openjdk@11.rb

```diff
@@ -1,10 +1,23 @@
 class OpenjdkAT11 < Formula
   desc "Development kit for the Java programming language"
   homepage "https://openjdk.java.net/"
-  url "https://github.com/openjdk/jdk11u/archive/refs/tags/jdk-11.0.12-ga.tar.gz"
-  sha256 "8c1022b6d59d3e68d6f2a431cb9a58fca550382844625649fa49623feef45996"
   license "GPL-2.0-only"
 
+  if Hardware::CPU.arm?
+    # Temporarily use build of unreleased openjdk 11.0.15 on Apple Silicon
+    url "https://github.com/openjdk/jdk11u/archive/refs/tags/jdk-11.0.15+5.tar.gz"
+    sha256 "c541dbc147a40b9cce987d4831970b96bb843471541bb25a140bbbfbd3e056de"
+    version "11.0.15"
```

Comment:
```suggestion
    version "11.0.14"
```

This will likely work better for bottle generation, like we did before: https://github.com/Homebrew/homebrew-core/blob/9f96b42c6250a265be75da7d8f57fd6761d6bcbb/Formula/openjdk.rb#L9

Reply:
11.0.14.1 actually judging by the URL.